replay_build_scripts/symbolication.mjs

```diff
@@ -9,15 +12,12 @@ export async function readSymbols(file, pdbFile) {
     }
     const textStart = getTextSectionAddress(pdbFile);
 
-    const process = spawn(`${__dirname}\\..\\..\\lib\\llvm-pdbutil.exe`, [
```

Comment:
I rewrote this code to be a bit more efficient and handle windows paths better. Sometimes these spawns would fail because the output would be too big.